package.json

```diff
@@ -54,6 +54,7 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "pretest": "gulp pretest",
+    "prestart": "webdriver-manager start > /dev/null 2>&1 &",
```

Comment:
Input redirection and backgrounding isn't going to work on windows unfortunately :(